src/components/OverviewPage/TransactionList.tsx

```diff
@@ -46,69 +35,29 @@ const OverviewPageTransactionList = ({ className, onTransactionClick }: Overview
   const { addresses, fetchAddressTransactionsNextPage, isLoadingData } = useAddressesContext()
   const totalNumberOfTransactions = addresses.map((address) => address.details.txNumber).reduce((a, b) => a + b, 0)
 
-  const allConfirmedTxs = addresses
-    .map((address) => address.transactions.confirmed.map((tx) => ({ ...tx, address })))
-    .flat()
-    .sort((a, b) => b.timestamp - a.timestamp)
-
-  const allPendingTxs = addresses
-    .map((address) => address.transactions.pending.map((tx) => ({ ...tx, address })))
-    .flat()
-    .sort((a, b) => b.timestamp - a.timestamp)
+  const allConfirmedTxs = getTransactionsForAddresses('confirmed', addresses)
+  const allPendingTxs = getTransactionsForAddresses('pending', addresses)
 
   const loadNextTransactionsPage = async () => {
     addresses.forEach((address) => fetchAddressTransactionsNextPage(address))
   }
 
   const showSkeletonLoading = isLoadingData && !allConfirmedTxs.length && !allPendingTxs.length
 
-  const transactionsTableHeadersI18n = transactionsTableHeaders.map((el) => ({ ...el, title: t(el.title) }))
```

Comment:
Does that mean we need to clean up our App.json from unused keys?

Reply:
Ah yeah, I missed one: `Direction`. The rest are still used by other components.